index.bs

```diff
@@ -1527,6 +1531,483 @@ longer be considered as sanitized.
+
+: `<a rel>`
+:: The `rel` attribute is not allowed, because it can claim ownership or relationship of the whole document.
+
+: `<a ping>`
+:: The `ping` attribute is not allowed, because it could notify an unintended third-party of the link being clicked.
+
+
+: `<audio>`, `<canvas>`, `<embed>`, `<iframe>`, `<img>`, `<picture>`, `<video>`
+:: These elements for embedding content are not allowed, because it leads to HTTP requests to arbitrary hosts.
+
+: `<math>` MathML math
+:: The reasoning for the inclusion of MathML is solely based on [[SafeMathML]].
+
+: `<svg>`
+:: This element is not allowed, because it leads to HTTP requests to arbitrary hosts.
+
+: `<form>`, `<label>`, `<input>`, `<fieldset>`, `<legend>`, `<textarea>`, `<button>`, `<select>`, `<option>`, `<output>`
+:: These form elements are disallowed, we do not expect them in user-controlled content.
+
+: `<dialog>`, `<details>`
+:: These interactive elements are disallowed, we do not expect them in user-controlled content.
+
+
+## Elements that are **never** allowed ## {#never-allowed}
+
+  <table>
+   <caption>List of unsafe and disallowed elements</caption>
+   <thead>
+    <tr>
+     <th> Element
+     <th> Namespace
+     <th> Description
+   <tbody>
+
+    <tr>
+     <td><code>embed</code></td>
+     <td>http://www.w3.org/1999/xhtml</td>
+     <td>This element is disallowed because it can include other, untrusted documents and including scripts.</td>
+    </tr>
+
+    <tr>
+     <td><code>frame</code></td>
+     <td>http://www.w3.org/1999/xhtml</td>
+     <td>This element is disallowed because it can include other, untrusted documents and including scripts.</td>
+    </tr>
+
+    <tr>
+     <td><code>iframe</code></td>
+     <td>http://www.w3.org/1999/xhtml</td>
+     <td>This element is disallowed because it can include other, untrusted documents and including scripts.</td>
+    </tr>
+
+    <tr>
+     <td><code>object</code></td>
+     <td>http://www.w3.org/1999/xhtml</td>
+     <td>This element is disallowed because it can include other, untrusted documents and including scripts.</td>
+    </tr>
+
+    <tr>
+     <td><code>script</code></td>
+     <td>http://www.w3.org/1999/xhtml</td>
+     <td>This element is disallowed because it executes scripts.</td>
+    </tr>
+
+    <tr>
+     <td><code>script</code></td>
+     <td>http://www.w3.org/2000/svg</td>
+     <td>This element is disallowed because it executes scripts.</td>
+    </tr>
+
+    <tr>
+     <td><code>use</code></td>
+     <td>http://www.w3.org/2000/svg</td>
+     <td>This element is disallowed because it can include other, untrusted documents and including scripts.</td>
+    </tr>
+  </table>
+
+
+## Event Handler Content Attributes
+
+All known event handler content attributes are disallowed, as they execute script.
+
+ <table>
+   <caption>List of all known and therefore disallowed event handler content attributes attributes</caption>
```

Comment:
I don't think we want to have yet another list for these. Let's use a reference instead.